lib/interactive_viewer.ml

```diff
@@ -47,14 +47,7 @@ let ui_of_operation operation =
       Ui.hcat
         [ W.string "Modification of "; W.string ~attr:blue_bold_attr path ]
 
-let ui_of_hunk hunk =
-  let line_to_string line =
-    match line with
-    | `Their text -> W.string ~attr:Notty.A.(fg red) text
-    | `Mine text -> W.string ~attr:Notty.A.(fg green) text
-    | `Common text -> W.string text
-  in
-  Ui.vcat @@ List.map line_to_string hunk.Patch.lines
```

Comment:
This diff is an artifact of being based off https://github.com/panglesd/diffcessible/pull/57, which has been reverted since. Could you rebase or `merge main` ?